instrumentation/github.com/aws/aws-lambda-go/otellambda/config.go

```diff
@@ -114,3 +129,10 @@ func WithPropagator(propagator propagation.TextMapPropagator) Option {
 		c.Propagator = propagator
 	})
 }
+
+// WithCustomAttributes configures a function that returns custom attributes.
+func WithCustomAttributes(eventAttrExtractor EventAttrExtractor) Option {
```

Comment:
These attributes are for traces. If we add metrics, this method name becomes misleading.
We should name it `WithTraceAttributesFn`.

See otelhttp for a precedent: https://pkg.go.dev/go.opentelemetry.io/contrib/instrumentation/net/http/otelhttp#WithMetricAttributesFn